app/assets/stylesheets/views/_home.scss

```diff
@@ -0,0 +1,80 @@
+.section-img {
+  background-image: image-url('landing-page/landing-page-1.jpg');
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: cover;
+  height: 35vh;
+  width: 100%;
+}
+
+main {
+  @include center-flex-column;
+  margin: 0 10%;
+  position: relative;
+  top: -10rem;
+  width: auto;
+
+  .search-options {
+    background-color: white;
+    border-radius: 10px;
+    box-shadow: 0px 0px 5px black;
```

Comment:
Same observation here, please use a variable to color black in your file _variables.scss